examples/PublishToMultipleQueuesRabbitMQ/NoOutputService.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Motor.Extensions.Hosting.Abstractions;
+using Motor.Extensions.Hosting.CloudEvents;
+using PublishToMultipleQueuesRabbitMQ.Model;
+
+namespace PublishToMultipleQueuesRabbitMQ;
+
+public class NoOutputService(
+    ITypedMessagePublisher<LeftMessage> leftPublisher,
+    ITypedMessagePublisher<RightMessage> rightPublisher)
+    : INoOutputService<InputMessage>
+{
+    // Handle incoming messages
+    private static int _messageCount = 0;
+
+    public Task<ProcessedMessageStatus> HandleMessageAsync(MotorCloudEvent<InputMessage> inputEvent, CancellationToken token = default)
+    {
+        // Get the input message from the cloud event
+        var input = inputEvent.TypedData;
+
+        // Sometimes we dont have anything to publish.
+        if (string.IsNullOrEmpty(input.FancyText))
+        {
+            // Might need to be logged, or published to some additional queue, or can simply be ignored. 
+            return Task.FromResult(ProcessedMessageStatus.Success);
+        }
+
+        _messageCount++;
+
+        // In all other cases we publish to the queue dictated by our business logic.
+        if (_messageCount % 2 == 0)
```

Comment:
I'm finally working on Motor.NET again and I just noticed that the static code analysis fails.

I don't know if the error is visible to you, but Codacy complains about accessing a static variable from a non-static context. That may be a bit nitpicky, but alternatively, you could just flip a coin to decide which publisher gets the message. That would still show the purpose of the example.